### PR TITLE
Feat: "Added Docs Page"

### PR DESCRIPTION
**Pull request by sadathshariff**
Made the layout for Docs Page and responsive sidebar

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -4,21 +4,65 @@
...
 }
 
 .navbar {
-  /* width: 100%; */
   padding: 2rem 1.2rem;
   display: flex;
   align-items: center;
   justify-content: space-between;
-  /* border: 1px solid red; */
+
```
Comment (rahulcs754) [inline CSS/style.css]: you leave this + symbols

File: CSS/style.css [hunk 2]
```diff
@@ -68,3 +110,72 @@
...
+  }
+}
+
+@media screen and (max-width: 800px) {
+  .side-nav {
+    /* visibility: hidden; */
+    display: none;
+  }
+  .show-navbar {
+    /* visibility: visible; */
```
Comment (rahulcs754) [inline CSS/style.css]: remove comment code from code

File: CSS/style.css [hunk 3]
```diff
@@ -68,3 +110,72 @@
...
+
+@media screen and (max-width: 600px) {
+  .container {
+    width: 100%;
+  }
+}
+
+@media screen and (max-width: 800px) {
+  .side-nav {
+    /* visibility: hidden; */
```
Comment (rahulcs754) [inline CSS/style.css]: remove comment code from code

File: CSS/style.css [hunk 4]
```diff
@@ -68,3 +110,72 @@
...
 }
+.example-code {
+  height: max-content;
+}
+
+.menu-div {
+  float: right;
+  visibility: hidden;
+}
+#menu-close {
```
Comment (rahulcs754) [inline CSS/style.css:133]: why you use id  there is any need.

**Review by rahulcs754 — COMMENTED** [review]
look good and please see changes

**Comment by sadathshariff** [comment]
> look good and please see changes

made the changes

File: CSS/style.css [hunk 5]
```diff
@@ -4,30 +4,79 @@
...
+  display: flex;
+  font-size: 1.2rem;
+  font-weight: 500;
+  justify-content: space-between;
+  margin-top: 3rem;
+}
+.mr-auto {
+  margin-right: auto;
+}
+.ml-auto {
```
Comment (krishanu2911) [inline CSS/style.css]: Just a small suggestion class name could mr-left-auto is easier to understand

**Review by krishanu2911 — COMMENTED** [review]
Just added small suggestion else seems good to go

File: CSS/style.css (hunk 5 above)
Comment (sadathshariff) [reply CSS/style.css]: made the changes

**Pull request merged**